dev/com.ibm.ws.security.utility/resources/com/ibm/ws/security/utility/resources/UtilityOptions.nlsprops

```diff
@@ -266,4 +266,50 @@ tlsProfiler.option-desc.file=\
 tlsProfiler.option-key.verbose=\
 \ \ \ \ --verbose
 tlsProfiler.option-desc.verbose=\
-\tInclude additional information about unsafe protocols.
\ No newline at end of file
+\tInclude additional information about unsafe protocols.
+
+# -------- importSigner OPTIONS ------------------------------------------------#
+# List of descriptive text for options. Use "\n\" for line continuations.
+# The code provides no whitespace: begin keys with "\ \ \ \ " for leading
+# whitespace, and begin descriptions with \t
+#------------------------------\n at 72 chars -------------------------\n\#
+#------------------------------\n at 72 chars Leading "\ \ \ \ "-------\n\#
+#------------------------------\n at 72 chars Leading "\t"-------------\n\#
+importSigner.desc=\
+\tImport the signer certificate of a target host and port into the provided keystore.
+importSigner.usage.options=\
+\t{0} importSigner [options]
+importSigner.option-key.password=\
+\ \ \ \ --password[=pwd]
+importSigner.option-desc.password=\
+\tKeystore password, minimum {1} characters.\n\
+\tIf no value is defined you will be prompted.\n\
```

Comment:
If no value is defined you will be prompted
-->
If no value is defined, you are prompted to specify a value